test/ansible/Makefile

```diff
@@ -0,0 +1,85 @@
+
```

Comment:
There's a whole bunch of Go-operator related code here.

Is the intention to fully rebuild this directory once the plugin can scaffold it with all of the ansible-specific stuff?